lib/ansible/modules/network/avi/avi_api_session.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+"""
+# Created on Aug 12, 2016
+#
+# @author: Gaurav Rastogi (grastogi@avinetworks.com) GitHub ID: grastogi23
+#
+# module_check: not supported
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+
+import json
+import time
+from ansible.module_utils.basic import AnsibleModule
+from copy import deepcopy
+
+HAS_AVI = True
+try:
+    from avi.sdk.avi_api import ApiSession, ObjectNotFound
+    from avi.sdk.utils.ansible_utils import (
```

Comment:
If these modules get into Ansible then is their any reason to maintain a copy in Galaxy? Having two sources for modules sounds like it could cause confusion for your modules.